R/checks.R

```diff
@@ -286,6 +290,24 @@ check_workflow <- function(x, pset = NULL, check_dials = FALSE) {
     }
   }
 
+  tune_tbl <- tune_args(x)
+
+  if (nrow(tune_tbl) > nrow(pset)) {
+    not_tunable <- tune_tbl$id[!tune_tbl$id %in% pset$id]
+    not_tunable_tbl <- tune_tbl[tune_tbl$id %in% not_tunable, ]
+    msg <-
+      c("!" = "{cli::qty(not_tunable)}The parameter{?s} {.var {not_tunable}}
+               {?was/were} marked with `tune()`, though {?is/are} not supported for tuning.")
+    if (all(not_tunable_tbl$source == "model_spec")) {
+      msg <-
+        c(msg,
+          "i" = "{cli::qty(not_tunable)}Have you supplied an engine that
+                 supports tuning {?this/these} parameter{?s}?")
```

Comment:
Can we avoid phrasing this as a question? It seems somewhat ambiguous because it can be interpreted either way

Reply:
Addressed in [84d08ed](https://github.com/tidymodels/tune/pull/549/commits/84d08ed54016169ebfd30597de11023801771849)!